src/pages/index.js

```diff
@@ -0,0 +1,349 @@
+import "./index.css";
+import { enableValidation, settings, resetValidation, disableButton } from "../scripts/validation.js";
+import { setButtonText } from "../utils/helpers.js";
+import Api from "../utils/Api.js";
+
+// import images
+import pencilSrc from "../images/pencil.svg";
+import whitePencilSrc from "../images/white-pencil.svg";
+import logoSrc from "../images/logo.svg";
+import plusSrc from "../images/plus.svg";
+import grayXSrc from "../images/gray-x.svg";
+import whitexSrc from "../images/white-x.svg";
+
+// initialize the api
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "c13b185c-8fbc-40ad-bb6b-9e8300eb8355",
+    "Content-Type": "application/json"
+  }
+});
+
+// modals
+const addCardModal = document.querySelector("#add-card-modal");
+const cardSubmitButton = addCardModal.querySelector(".modal__submit-button");
+const addCardElement = addCardModal.querySelector(".modal__form");
```

Comment:
**COULD BE IMPROVED**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms